### PR TITLE
rename customer-driven value to customer-first, elaborate on it

### PR DESCRIPTION
**Pull request by sqs**
I renamed our customer-driven value to customer-first and wrote more about it, including points that have come up in conversations with team members that they or I have found helpful in understanding what this value means and why it's important.

All feedback is very much appreciated!

- Renaming "customer-driven" to "customer-first". Why? It's simpler, feels more natural (evidence: the [customer-first page](https://handbook.sourcegraph.com/company/customer-first) was called customer-first not customer-driven), and better captures our intended definition (we wanted "driven" to mean that we are driven to serve customer's interests, but I guess it could be misunderstood to mean that we let customers directly tell us what to build, or something).
- Explain why we are customer-first (so we can become customer-funded).
- Say what we are NOT when we choose to be customer-first.
- Say what being customer-first means for people in various roles (which I have found to resonate in conversations with team members and in interviews).
- Call out the distinction between customer-first and prospect-first. I believe we should strive to be more customer-first and less prospect-driven. I've spoken with many team members about this and the risk of building features for orgs that aren't yet customers; everyone agrees and nobody intended to be prospect-driven, but it emerged unintentionally from our strategy and goals over the last year. It's sometimes necessary, but it's risky, and we should find ways (in strategy and execution) to iterate with existing customers whenever possible. Having this distinction in our values will be a useful reminder and communication tool to help nudge us in the right way. Our [FY22Q4 company-wide OKRs](https://docs.google.com/document/d/1FjKdlTFpcWfDF3zMa-SkJRemWHJAinECpYLfT3X92OU/edit#) talk about this as well, and I'll refer back to the new text on the values page a lot as we write down our future strategy.

### CODE REVIEW

File: content/company/values.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 These values are the beliefs and principles that help us achieve our [goals](goals/index.md) and [strategy](../company/strategy/index.md) and build an [inclusive team](../communication/code_of_conduct.md). These apply at every level of our organization—"you" refers to individuals, teams, and Sourcegraph as a whole.
 
-- [**Be customer driven**](#be-customer-driven): You proactively work on the right things by continuously orienting your goals around delivering value to your customers.
+- [**Be customer driven**](#be-customer-driven): You earn and keep the trust of our customers by putting their interests first (ahead of Sourcegraph's interests and your team's interests).
```
Comment (jeanduplessis) [inline content/company/values.md]:
```suggestion
- [**Be customer-driven**](#be-customer-driven): You earn and keep the trust of our customers by putting their interests first.
```

**suggest (non-blocking)**: Adding the part around putting customer interest ahead of Sourcegraph's interest in the main value tagline doesn't feel right. It does feel right to have it in the descriptive text of the value as we can provide better context as to what we mean with this.

**Review by jeanduplessis — APPROVED** [review]
❤️  Love this.

**Comment by felixfbecker** [comment]
@sqs did you consider linking to https://handbook.sourcegraph.com/company/customer-first ? I feel like there's some overlap here in explaining what customer-first means and why we are a customer-first company 🙂

**Comment by sqs** [comment]
@felixfbecker Ah, I had forgotten about that page. That brought up another change I wanted to make and, given that page, decided to go ahead and make it:

Renaming "customer-driven" to "customer-first". Why? It's simpler, feels more natural (evidence: the customer-first page was called customer-first not customer-driven), and better captures our intended definition (we wanted "driven" to mean that we are driven to serve customer's interests, but I guess it could be misunderstood to mean that we let customers directly tell us what to build, or something).

File: content/company/remote/remote_hackathons.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ For many group sizes/dynamics, it might make sense to set up separate slack chan
 
 ### Share that you're doing a hackathon!
 
-Hackathons encompass many of the Sourcegraph values – high agency, working as a team, being customer driven, continuous learning – and you should share that you're doing a hackathon with the whole company! (Plus, if you share it, then you're adding "communicating openly and transparently"!)
+Hackathons encompass many of the Sourcegraph values – high agency, working as a team, being customer first, continuous learning – and you should share that you're doing a hackathon with the whole company! (Plus, if you share it, then you're adding "communicating openly and transparently"!)
```
Comment (jeanduplessis) [inline content/company/remote/remote_hackathons.md]:
```suggestion
Hackathons encompass many of the Sourcegraph values – high agency, working as a team, being customer-first, continuous learning – and you should share that you're doing a hackathon with the whole company! (Plus, if you share it, then you're adding "communicating openly and transparently"!)
```

File: content/marketing/content/index.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ How our content strategy aligns with our company values.
...
 | -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Be customer driven   | Our content serves our customers’ interests first. We focus on providing valuable information to our audiences over pitching or selling our product.                                                                                                                                                                         |
+| Be customer first    | Our content serves our customers’ interests first. We focus on providing valuable information to our audiences over pitching or selling our product.                                                                                                                                                                         |
```
Comment (jeanduplessis) [inline content/marketing/content/index.md]:
```suggestion
| Be customer-first    | Our content serves our customers’ interests first. We focus on providing valuable information to our audiences over pitching or selling our product.                                                                                                                                                                         |
```

**Review by jeanduplessis — COMMENTED** [review]
Consistently hyphenate customer-first

File: content/company/values.md [hunk 4]
```diff
@@ -14,9 +14,15 @@ These values are the beliefs and principles that help us achieve our [goals](goa
 
 ## Values in detail
 
-### Be customer driven
+### Customer-first
 
-You proactively work on the right things by continuously orienting your goals around delivering value to your customers.
+You earn and keep the trust of our customers by putting their interests first, ahead of Sourcegraph's interests and your team's interests. Over the long term, we need to be a customer-funded business, and you'll help us get there by being customer-first.
+
+- We are not engineering-driven. We are not sales-driven. Being customer-first unites us all (engineering, sales, and every other team).
```
Comment (CristinaBirkel) [inline content/company/values.md]:
```suggestion
- We are not engineering-first. We are not sales-first. Being customer-first unites us all (engineering, sales, and every other team).
```
I think aligning these gives them a stronger relation to "customer-first," and helps the idea resonate more. Just a thought!

Comment (sqs) [reply content/company/values.md]: @CristinaBirkel Yes, this is a good improvement. Thank you!

**Pull request merged**